lib/fetch/sanitize.js

```diff
@@ -124,3 +124,23 @@ export function sanitizeTicket(json) {
     available: get('ticket.available'),
   };
 }
+
+export function sanitizeBadger(json) {
+  const get = makeGetter(json);
+  return {
+    id: json.id,
+    firstName: get('badger.first-name'),
+    lastName: get('badger.last-name'),
+    jobTitle: get('badger.job-title'),
+    imageUrl: get('badger.image-url'),
+    startDate: get('badger.start-date'),
+    about: get('badger.about'),
+    skills: get('badger.skills'),
+    influence: get('badger.influence'),
```

Comment:
Hmmm, that depends if everyone will be required to do add those or not. But yes, I suppose that could happen to those fields and maybe even others such as github url, for example. In this case some modifications are necessary! Hold on :)